README.md

```diff
@@ -45,7 +45,7 @@ Ethereum (batches or validator set updates).
 ```shell
 $ umeed tx gravity set-orchestrator-address \
   {validatorAddress} \
-  {validatorAddress} \
+  {orchestrator-address} \
```

Comment:
Let's be consistent with our usage here. I'd prefer this to be `<orchestrator-address>` along with updating the remaining flags too

Reply:
@alexanderbez  do you mean like this? 

```shell
$ umeed tx gravity set-orchestrator-address \
  <validatorAddress> \
  <orchestrator-address> \
  <ethAddress> \
  --eth-priv-key="..." \
  --chain-id="..." \
  --fees="..." \
  --keyring-backend=... \
  --keyring-dir=... \
  --from=...
```